app/models/manageiq/providers/azure/inventory/persister/target_collection.rb

```diff
@@ -16,7 +16,7 @@ def initialize_inventory_collections
     # Child models with references in the Parent InventoryCollections for Cloud
     add_inventory_collections(
       cloud,
-      %i(hardwares operating_systems networks disks vm_and_template_labels vm_and_template_taggings
```

Comment:
why were vm_and_template_labels vm_and_template_taggings removed?

Reply:
we do not do those for azure, correct? I didn't see it

Reply:
Ah true, Alex is implementing this right now so we can add these back once his work is complete.

Reply:
ok, will that be also backported?

Reply:
yea